src/grpc.ts

```diff
@@ -62,20 +62,18 @@ const COMMON_PROTO_DIRS = [
   'protobuf',  // This is an additional path that the common protos depend on.
   'rpc',
   'type',
-];
-
-const COMMON_PROTO_GLOB_PATTERNS = COMMON_PROTO_DIRS.map(dir => {
-  return path.join(googleProtoFilesDir, 'google', dir, '**', '*.proto');
-});
-
-const COMMON_PROTO_FILES =
-    globby.sync(COMMON_PROTO_GLOB_PATTERNS)
-        .map(filename => {
-          return path.normalize(filename);
-        })
-        .map(filename => {
-          return filename.substring(googleProtoFilesDir.length + 1);
-        });
+].map(dir => path.join(googleProtoFilesDir, 'google', dir));
+
+
+const COMMON_PROTO_FILES = COMMON_PROTO_DIRS
+                               .map(dir => {
+                                 return (walk.sync(dir) as string[])
+                                     .filter(f => path.extname(f) === '.proto')
+                                     .map(
+                                         f => path.normalize(f).substring(
+                                             googleProtoFilesDir.length + 1));
+                               })
+                               .reduce((a, c) => a.concat(c), []);
```

Comment:
I would hope that future PRs can incrementally: convert it to an async walk; convert COMMON_PROTO_FILES to a promise, making it lazy; or ideally, remove the need to scan directories to begin with.